src/renderer/browser/components/RequestsProcessor/ClaimGas/index.js

```diff
@@ -34,7 +34,7 @@ export default function makeClaimComponent(claimActions) {
       wif
     })),
     withNullLoader(claimActions),
-    withRejectMessage(claimActions, 'Could not claim GAS.'),
+    withRejectMessage(claimActions, ({ error }) => (`Could not claim GAS.\n${error}`)),
```

Comment:
Can you change all of these to use a colon (`:`) to separate the hardcoded message from the interpolated error message instead?  Then this will be consistent with our error message formatting everywhere else in the app.  i.e.:

```js
`Could not claim GAS: ${error}`
```

Reply:
done